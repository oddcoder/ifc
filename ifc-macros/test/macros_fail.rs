use ifc_macros::ifc_block;

fn main() {
    ifc_block!{
        let x = 5;
        let _y = format!("{}", x);
    }
}