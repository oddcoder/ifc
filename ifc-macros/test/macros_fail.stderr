error: Cannot pass high expression as an argument to functions or macros
 --> $DIR/macros_fail.rs:6:18
  |
6 |         let _y = format!("{}", x);
  |                  ^^^^^^^^^^^^^^^^
  |
help: High Expression
 --> $DIR/macros_fail.rs:6:32
  |
6 |         let _y = format!("{}", x);
  |                                ^
